publish.rb

```diff
@@ -31,7 +31,7 @@
     FileUtils.cp_r "_site/.", tmp
     Dir.chdir tmp
     system "git init"
-
+    
```

Comment:
git complains about trailing whitespace of any sort, so I prefer to never have trailing whitespace if at all possible. All of the editors I use nowadays are smart enough that they will indent correctly without having to leave whitespace on empty lines like this.